index.html

```diff
@@ -0,0 +1,290 @@
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+          </ul>
+          <div class="action2">
+            <p>See Project</p>
+          </div>
+          </div>
+      </div>  
+      <div class="card-work">
+        <div class="card-image">
+          <img src="./images/4.png" alt="card-work">
+          <h2 class="h2">
+            Multi-Post
+            Stories
+          </h2>
+        </div> 
+        <div class="left4-block">
+            <div class="frame4">
+              <p>CONOPY</p>
+              <img src="./images/Counter.jpg" alt="">
+              <p class="p1">Back End Dev</p>
+              <img src="./images/Counter.jpg" alt="">
+              <p class="p1">2020</p>
+            </div>
+        <p class="primary4">
+          A daily selection of privately<br />personalized reads; 
+          no accounts or<br /> sign-ups required.
+      </p>
+     <ul>
+        <li><a href="#">html</a></li>
+        <li><a href="#">css</a></li>
+        <li><a href="#">javascript</a></li>
+      </ul>
+      <div class="action3">
+        <p>See Project</p>
+      </div>
+    </div>
+        </div>
+        
+    </section>
+      <section class="about"> 
+      <div class="top">
+        <h2 class="h2">About<br />Myself</h2>
+        <div class="primary5">
+          <p>
+            Hello I am a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, do not hesitate to contact me.
+          </p>
+        </div>
+        <div class="connect"> 
+          <p>LET US CONNECT</p>
+        </div>
+        <div class=" social_links1">
+          <ul>
+            <li>
+              <a href="#">
+                <img class="twit1-icon" src="./images/twitter.png" alt="twit-icon">
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img class="git1-icon" src="./images/github.png" alt="git-icon">
+              </a>
+            </li>
+            
+            <li>
+              <a href="#">
+                <img class="ln1-icon" src="./images/Linkedin-icon.png" alt="ln-icon">
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img class="m1-icon" src="./images/medium.png" alt="m-icon">
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img class="a1-icon" src="./images/angellist.png" alt="a-icon">
+              </a>
+            </li>
+           </ul>
+          
+        </div>
+        <div class="action4">
+          <p>Get my Resume</p>
+        </div>
+      </div>
+      <div class="skill-list">
+        <div class="frame12">
+          <div class="language">
+            <span class="text">Languages</span>
+            <span class="down-angle"><i class="fa-solid fa-angle-down"></i></span>
```

Comment:
- [ ] Kindly wrap this `languages` with ul and put this `language` in li tag for better practice